src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -206,49 +206,59 @@ private static IEnumerable<(FunctionOverload functionOverload, ResourceScope all
             yield return (
                 new FunctionOverloadBuilder("tenant")
                     .WithDynamicReturnType(GetTenantReturnType, new TenantScopeType(Enumerable.Empty<FunctionArgumentSyntax>(), Enumerable.Empty<TypeProperty>()))
+                    .WithGenericDescription("Returns the current tenant scope.")
                     .WithDescription("Returns the current tenant scope.")
                     .Build(),
                 ResourceScope.Tenant | ResourceScope.ManagementGroup | ResourceScope.Subscription | ResourceScope.ResourceGroup);
 
+            const string managementGroupGenericDescription = "Returns a management group scope.";
```

Comment:
>Returns a management group scope. [](http://example.com/codeflow?start=62&length=33)

From docs: Returns an object with properties from the management group in the current deployment.